src/webhooks/registry.ts

```diff
@@ -297,7 +313,7 @@ const WebhooksRegistry: RegistryInterface = {
 
         if (ShopifyUtilities.safeCompare(generatedHash, hmac as string)) {
           const graphqlTopic = (topic as string).toUpperCase().replace(/\//g, '_');
-          const webhookEntry = WebhooksRegistry.webhookRegistry.find((entry) => entry.topic === graphqlTopic);
+          const webhookEntry = WebhooksRegistry.webhookRegistry[graphqlTopic];
```

Comment:
Should this be a call to `getHandler` instead?

Reply:
yess